### DISCUSSION TITLE
Shipwright Community Summit 2022 - Q&A

### DISCUSSION THREAD

**Discussion by qu1queee** [shipwright-io/community#89] [Q&A]
We are pleased to announce that this year we will be hosting our first Shipwright **Community Summit** at the [cdCon 2022](https://cdcon2022.sched.com/) Conference.

We will use this Discussion as a Q&A to enable anyone interested on participating in the Community Summit to drop any sort of questions in advance. We will revisit each question during the Community Summit.

For more information on how to participate in the Summit, please refer to [here](https://shipwright.io/blog/2022/05/23/upcoming-shipwright-community-summit/).

**Comment by adambkaplan** [comment]
Closing as completed
